etc/skel/.config/hypr/config/keybinds.conf

```diff
@@ -28,8 +28,8 @@ bindd = $mainMod, G, Remove gaps between window, exec, hyprctl --batch "keyword
 
 # ======= Volume Control =======
 
-bindde = , XF86AudioLowerVolume, Decreases player audio volume, exec, pamixer -ud 3 && pamixer --get-volume > /tmp/$HYPRLAND_INSTANCE_SIGNATURE.wob
-bindde = , XF86AudioRaiseVolume, Increases player audio volume, exec, pamixer -ui 3 && pamixer --get-volume > /tmp/$HYPRLAND_INSTANCE_SIGNATURE.wob
```

Comment:
I believe `bindde` is the correct bind to use here, since we want it to keep repeating itself when its held.

Reply:
I agree